smaller_constraints.py

```diff
@@ -0,0 +1,41 @@
+from ortools.sat.python import cp_model
+from ortools.sat.python.cp_model import IntVar
+import datetime
+
+from models.employees.employee import Employee
+from models.shifts.shift import Shift
+from models.shifts.shifts_types_enum import ShiftTypesEnum
```

Comment:
where is the code in main that would call this code? i want to see how it would be used